framework/platform/bpl/include/bpl/bpl_err.h

```diff
@@ -27,69 +27,71 @@ extern "C" {
+    ERROR_CODE(BPL_ERR_SLAVE_WIFI_CREDENTIALS_SET_FAILED)                                          \
+    ERROR_CODE(BPL_ERR_SLAVE_POST_INIT_CONFIG_FAILED)                                              \
+    ERROR_CODE(BPL_ERR_SLAVE_TX_CHANGE_STATE_FAILED)                                               \
+    ERROR_CODE(BPL_ERR_SLAVE_UPDATE_CREDENTIALS_FAILED)                                            \
+    ERROR_CODE(BPL_ERR_SLAVE_FAILED_CONNECT_TO_PLATFORM_MANAGER)                                   \
+    ERROR_CODE(BPL_ERR_SLAVE_PLATFORM_MANAGER_REGISTER_TIMEOUT)                                    \
+    ERROR_CODE(BPL_ERR_SLAVE_SLAVE_BACKHAUL_MANAGER_DISCONNECTED)                                  \
+    ERROR_CODE(BPL_ERR_SLAVE_STOPPED)                                                              \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_START)                                                           \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_DISCONNECTED)                                                    \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_HOSTAP_DISABLED)                                                 \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_ATTACH_FAIL)                                                     \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_SUDDEN_DETACH)                                                   \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_HAL_DISCONNECTED)                                                \
+    ERROR_CODE(BPL_ERR_AP_MANAGER_CAC_TIMEOUT)                                                     \
+    ERROR_CODE(BPL_ERR_MONITOR_DISCONNECTED)                                                       \
+    ERROR_CODE(BPL_ERR_MONITOR_HOSTAP_DISABLED)                                                    \
+    ERROR_CODE(BPL_ERR_MONITOR_ATTACH_FAIL)                                                        \
+    ERROR_CODE(BPL_ERR_MONITOR_SUDDEN_DETACH)                                                      \
+    ERROR_CODE(BPL_ERR_MONITOR_HAL_DISCONNECTED)                                                   \
+    ERROR_CODE(BPL_ERR_MONITOR_REPORT_PROCESS_FAIL)                                                \
+    ERROR_CODE(BPL_ERR_CONFIG_PLATFORM_REPORTED_INVALID_CONFIGURATION)                             \
+    ERROR_CODE(BPL_ERR_CONFIG_BACKHAUL_WIRED_INTERFACE_IS_UNSUPPORTED)                             \
+    ERROR_CODE(BPL_ERR_CONFIG_BACKHAUL_WIRELESS_INTERFACE_IS_UNSUPPORTED)                          \
+    ERROR_CODE(BPL_ERR_CONFIG_NO_VALID_BACKHAUL_INTERFACE)                                         \
+    ERROR_CODE(BPL_ERR_CONFIG_BRIDGE_INTERFACE_IS_EMPTY)                                           \
+    ERROR_CODE(BPL_ERR_WIFI_CONFIGURATION_CHANGE_TIMEOUT)                                          \
+    ERROR_CODE(BPL_ERR_WATCHDOG_PROCESS_STUCK)                                                     \
+    ERROR_CODE(BPL_ERR_WATCHDOG_PROCESS_ZOMBIE)                                                    \
+    ERROR_CODE(BPL_ERR_LAST)
```

Comment:
As part of my changes on remove non-passive mode code, I deleted a lot of unused BPL error since it is not being used anymore, and aligned it as you did. Please add the new error codes at the end of the list so it would be easy to rebase.

Reply:
The new error codes values have to be 1 and 2 so they can't be added to the end of the list. Rebase shouldn't be too hard even as it is now.